src/playbook.yaml

```diff
@@ -95,3 +95,11 @@
 
     - name: Install BiocManager
       shell: "R -e \"install.packages('BiocManager')\""
+
+    - name: update limits.conf
+      shell:
+        cmd:
+          # The ">>" commands won't work unless we make this permission change
+          sudo chmod 666 /etc/security/limits.conf
+          echo '* soft nofile 1000000' >> /etc/security/limits.conf
```

Comment:
ie, `sudo echo '* soft nofile 1000000' >> /etc/security/limits.conf`

Reply:
@ahayden That's a good idea but it didn't work when I tried it on an RStudio instance. I.e., I ran "sudo echo ... >> /etc/security/limits.conf" and it didn't work. But once I changed the permissions the command did work (without 'sudo', i.e. as ssmuser).